log-manager/src/main/java/io/airlift/log/RollingFileMessageOutput.java

```diff
@@ -403,7 +264,7 @@ private synchronized void rollFile()
             try {
                 newFileName = LogFileName.generateNextLogFileName(symlink, compressionType.getExtension());
                 newFile = symlink.resolveSibling(newFileName.getFileName());
-                newOutputStream = new BufferedOutputStream(Files.newOutputStream(newFile, CREATE_NEW), MAX_BATCH_BYTES);
+                newOutputStream = Files.newOutputStream(newFile, CREATE_NEW);
```

Comment:
I reverted some changes, but I am now using the suppressed exceptions in a few places to avoid having to use the ErrorManager because we're not extending handler anymore.